environments/java_codegen_monitoring/Makefile

```diff
@@ -2,7 +2,7 @@ SHELL = /bin/sh
 
 NS := datarobot-oss
 IMAGE_NAME := mlops-sap-monitoring-scoring-code
-VERSION := 1.0.0
+VERSION := 1.0.1
```

Comment:
If this was split into a dedicated file, I'm pretty sure you can use a shell call with cat to slurp in the version.